Parameters/Parameters.pt-BR.json

```diff
@@ -0,0 +1,46 @@
+{
+  "AddFanCurve": "Adicionar curva de ventilador",
+  "AdvancedHysteresis": "Histerese avançada",
+  "Average": "Média",
+  "Deadband": "Faixa morta",
+  "DeadbandDescription": "Intervalo abaixo da temperatura de carga onde a curva não mudará, a menos que a temperatura esteja subindo.",
+  "FanSpeed": "Velocidade do ventilador",
+  "FileName": "Nome do arquivo",
+  "Function": "Função",
+  "Hysteresis": "Histerese",
+  "HysteresisDescription": "Diferença mínima de temperatura para que uma mudança ocorra.",
+  "HysteresisDownOnly": "Histerese só se aplica na descida",
+  "IdleSpeed": "Velocidade em repouso",
+  "IdleTemp": "Temperatura em repouso",
+  "IgnoreHysteresis": "Ignorar histerese nas temperaturas mínima e máxima",
+  "LoadSpeed": "Velocidade de carga",
+  "LoadTemp": "Temperatura de carga",
+  "Max": "Máximo",
+  "MaxSpeed": "Velocidade máxima",
+  "MaxTemp": "Temperatura máxima",
+  "Min": "Mínimo",
+  "MinSpeed": "Velocidade mínima",
+  "MinTemp": "Temperatura mínima",
+  "Offset": "Deslocamento",
+  "OffsetDescription": "Deslocamento em % a ser aplicado ao controle selecionado",
+  "OffsetTemperatureDescription": "Deslocamento em graus ou percentual (proporcional) a ser aplicado.",
+  "Path": "Caminho",
+  "PathDescription": "Caminho do arquivo do sensor.",
+  "ProportionalOffset": "Deslocamento Proporcional",
+  "ProportionalOffsetDescription": "O deslocamento deve ser multiplicado proporcionalmente em vez de adicionado",
+  "ResponseTime": "Tempo de resposta",
+  "ResponseTimeDescription": "Tempo mínimo antes que uma mudança possa ocorrer.",
+  "RpmMode": "Modo RPM",
+  "SelectControl": "Selecionar um controle",
+  "SelectedControl": "Controle selecionado",
+  "SelectedControlDescription": "O controle para sincronizar",
+  "Step": "Passo",
+  "StepDescription": "Taxa em que o valor mudará após o tempo de resposta dado. O passo é a metade na descida.",
+  "Subtract": "Subtrair",
+  "Sum": "Somar",
```

Comment:
```suggestion
  "Sum": "Soma",
```

Reply:
This could change depending on context, would need to evaluate the use within the software

Reply:
It's a Mix sensor, so it's Somar as a function